homeassistant/components/opentherm_gw/__init__.py

```diff
@@ -416,27 +391,27 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 class OpenThermGatewayDevice:
     """OpenTherm Gateway device class."""
 
-    def __init__(self, hass, config_entry):
+    def __init__(self, hass: HomeAssistant, config_entry: ConfigEntry) -> None:
         """Initialize the OpenTherm Gateway."""
         self.hass = hass
         self.device_path = config_entry.data[CONF_DEVICE]
         self.gw_id = config_entry.data[CONF_ID]
         self.name = config_entry.data[CONF_NAME]
         self.climate_config = config_entry.options
         self.config_entry_id = config_entry.entry_id
-        self.status = {}
+        self.status: dict[str, Any] = {}
```

Comment:
dict[str, Any] to be checked more thoroughly

Reply:
It should be something like this (untested and I'm not very good with type hints):
`dict[str, dict[str, Any]]`